packages/web/src/scss/components/SegmentedControl/_tools.scss

```diff
@@ -0,0 +1,51 @@
+@use 'sass:map';
+@use 'theme';
+
+@mixin generate-segmented-control-variant($variant, $config) {
```

Comment:
Thinking aloud: this code looks quite similar to our `generate-*()` mixins in `scss/tools/dictionaries`. If we agreed that "filled" and "outline" are a new "Variant" dictionary, we could:

a. rewrite this mixin to use custom properties,
b. move it to `scss/tools/dictionaries` to make it reusable for other components using the "Variant" dictionary – our form fields might be the next.

I will take it to the Long Topics.